### PR TITLE
kelly/fix_spacing

### PR DESCRIPTION
**Pull request by kellybinary**

### CODE REVIEW

File: src/templates/app/_includes/pep_declaration.jsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ const PepDeclaration = () => (
         <div className='gr-padding-10 gr-12'>
             <input id='not_pep' type='checkbox' />
             <label htmlFor='not_pep'>
-                {it.L('I acknowledge that I am not a politically exposed person (PEP).')}
+                {it.L('I acknowledge that I am not a politically exposed person (PEP).')}{' '}
```
Comment (negarn) [inline src/templates/app/_includes/pep_declaration.jsx]: why not add `&nbsp` instead of a literal empty space?

Comment (kellybinary) [reply src/templates/app/_includes/pep_declaration.jsx]: i thought `{' '}` would be more consistent with jsx syntax, but sure we can use `&nbsp;` if that's preferred

Comment (negarn) [reply src/templates/app/_includes/pep_declaration.jsx]: up to you, I guess :) there are more inconsistent things in our code than spacing convention.

Comment (kellybinary) [reply src/templates/app/_includes/pep_declaration.jsx]: hmm, better to stick to one thing, i'll change it to `&nbsp;` :P

**Pull request merged**